### DISCUSSION TITLE
Define and execute command

### DISCUSSION THREAD

**Discussion by Kingdom-Of-Heaven** [Eugeny/tabby#3615] [General]
Hi. I would like to define some command and then be able to chooose it from list when neded to be executed in terminal. I found nice option in SSH connections > scripts- this one is nice nevertheless i am obligated to create ssh connection as well which i do not need. How to do so without defining ssh connection so just to define command?

**Comment by Eugeny** [comment]
Check out the `quick-cmds` plugin.

**Reply by Kingdom-Of-Heaven** [reply]
Seems to be outdated but could you simply tell me how to define and save command in that lib?

**Reply by Eugeny** [reply]
Just install the quick-cmds plugin and add your commands under Settings -> Quick Commands. You can then either trigger them from a toolbar button or a hotkey.
